### PR TITLE
Create WOTMotion.es.vtt

### PR DESCRIPTION
**Pull request by AndreaCimminoArriaga**
Spanish translation of the subtitles

### CODE REVIEW

File: videos/WoT-Explainer-Video/WOTMotion.es.vtt [hunk 1]
```diff
@@ -0,0 +1,279 @@
...
+41
+00:02:47.500 --> 00:02:51.000
+los distintos modelos de datos
+y protocolos de los dispositivos o servicios.
+
+
+42
+00:02:51.500 --> 00:02:57.000
+Sin embargo, con WoT los desarrolladores crean
+soluciones IoT en un par de iteraciones
```
Comment (agmangas) [inline videos/WoT-Explainer-Video/WOTMotion.es.vtt:234]:
```suggestion
soluciones IoT en unas pocas iteraciones
```

File: videos/WoT-Explainer-Video/WOTMotion.es.vtt [hunk 2]
```diff
@@ -0,0 +1,279 @@
...
+
+48
+00:03:17.500 --> 00:03:22.000
+Utilizando WoT el proyecto IoT del fruticultor
+puede llevarse a cabo fácilmente.
+
+49
+00:03:22.500 --> 00:03:27.000
+Ahora los desarrolladores están listos
+para llevar a cabo cualquier reto IoT.
```
Comment (agmangas) [inline videos/WoT-Explainer-Video/WOTMotion.es.vtt:273]:
```suggestion
para llevar a cabo su siguiente reto IoT.
```

A bit more consistent with the original meaning maybe?

File: videos/WoT-Explainer-Video/WOTMotion.es.vtt [hunk 3]
```diff
@@ -0,0 +1,279 @@
...
+
+50
+00:03:27.500 --> 00:03:41.000
+y sí, WoT es la metodología
+que usarán!
```
Comment (agmangas) [inline videos/WoT-Explainer-Video/WOTMotion.es.vtt:278]:
```suggestion
¡Y sí, WoT es LA metodología
que usarán!
```

The original is in caps.

File: videos/WoT-Explainer-Video/WOTMotion.es.vtt [hunk 4]
```diff
@@ -0,0 +1,279 @@
+WEBVTT
+
+1
+00:00:00.500 --> 00:00:03.000
+Un fruticultor esta pensando formas
```
Comment (agmangas) [inline videos/WoT-Explainer-Video/WOTMotion.es.vtt:5]:
```suggestion
Un fruticultor está pensando formas
```

File: videos/WoT-Explainer-Video/WOTMotion.es.vtt [hunk 5]
```diff
@@ -0,0 +1,279 @@
...
+
+12
+00:00:41.500 --> 00:00:45.000
+Sin embargo, están preocupados por posibles
+problemas derivados de la interoperabilidad
+
+13
+00:00:45.500 --> 00:00:49.000
+algunos de los cuales ya experimentaron 
+en proyectos IoT anteriores
```
Comment (agmangas) [inline videos/WoT-Explainer-Video/WOTMotion.es.vtt:71]:
```suggestion
en proyectos IoT anteriores.
```

**Review by agmangas — APPROVED** [review]
@AndreaCimminoArriaga feel free to reject any changes 😄.

I tried to sound a bit more informal and maintain consistency with the original english subs.

File: videos/WoT-Explainer-Video/WOTMotion.es.vtt [hunk 6]
```diff
@@ -0,0 +1,279 @@
...
+
+18
+00:01:08.500 --> 00:01:12.000
+WoT puede ser aplicado en
+un gran número de campos IoT,
+
+19
+00:01:12.500 --> 00:01:15.000
+como la automatización en 
+fabricas o edificios,
```
[the remark this answers is not in the document]
Comment (AndreaCimminoArriaga) [reply videos/WoT-Explainer-Video/WOTMotion.es.vtt:104]: Por qué mayúsculas ?

**Comment by AndreaCimminoArriaga** [comment]
Hi @agmangas, may thanks for the feedback and corrections. I think all of them, except the one of the upper case, where more than correct and helpful. For sure they enriched the translation, therefore, many thanks

**Comment by agmangas** [comment]
Thanks to you for doing the translation in the first place 🙇

**Comment by sebastiankb** [comment]
muchas gracias @AndreaCimminoArriaga y @agmangas. 

The PR is ready for merging, right?

**Comment by sebastiankb** [comment]
from today's marketing call we decided to merge.

Thanks again  @AndreaCimminoArriaga and @agmangas

**Pull request merged**